### DISCUSSION TITLE
Is there any set release date?

### DISCUSSION THREAD

**Discussion by karel-suchomel-ed** [gluestack/gluestack-ui#668] [General]
Hello,
First of all, great job with this lib. I am already looking forward to working with it.
Currently, in my production app, which is pretty huge, I'm using NativeBase, and I want to migrate over to GlueStack as soon as possible. Is there any set release date for this lib? Or at least approximation.

Thanks.

**Comment by Ali-Aref** [comment]
First of all thank you for your great work!

Same here, my app is almost at the publish stage with nativeBase, but I want to migrate to the glueStack. I have two questions.
1. Is there any release data for this ?
2. Any ways to migrate from nativebase to this? I know the components are almost the same and I converted them manually, anyways I would like to hear from you guys.
